### ISSUE TITLE
Seems like v1.9.7 has broken builtin binding

### ISSUE THREAD

**Issue by allnetru** [Restream/reindexer#19]
To reproduce run example usage from readme.md twice (!) and you'll get:
```
panic: Reindex binding 'builtin' init error: rq:Namespace '#config' does not exist

goroutine 1 [running]:
github.com/restream/reindexer.NewReindex(0xc42019e180, 0x23, 0x0, 0x0, 0x0, 0xc42019e180)
       /home/biggieman/go/src/github.com/restream/reindexer/reindexer.go:155 +0x793
```

v1.9.6 works fine by the way

**Comment by allnetru** [comment]
Fixed in 1.10. Thanks!

**Comment by olegator77** [comment]
Yes, this issue was fixed in 1.10. 
Thank you for bug report
